tests/neo4j/test_bookmarks.py

```diff
@@ -72,3 +72,136 @@ def test_no_bookmark_after_rollback(self):
 
         bookmarks = self._session.last_bookmarks()
         self.assertEqual(len(bookmarks), 0)
+
+    @cluster_unsafe_test
+    def test_fails_on_invalid_bookmark(self):
```

Comment:
Do we want to have a cluster safe version of this using tx func?

Reply:
I've added `test_fails_on_invalid_bookmark_using_tx_func`.